src/hooks/reverseGeocode.ts

```diff
@@ -11,7 +11,7 @@ export const useReverseGeocode = () => {
       position,
       setValue
     }: {
-      areaServiceData: { postalCodes: string[] } | undefined;
+      areaServiceData?: { postalCodes?: string[] };
```

Comment:
does not relate to this commit, right?

Reply:
No. That was a minor type update. That's why I didn't add it to the commit.